src/pytest_postgresql/executor.py

```diff
@@ -176,10 +176,12 @@ def wait_for_postgres(self):
         start_info = 'database system is ready to accept connections'
         # wait for expected message.
         while 1:
-            with open(self.logfile, 'r') as content_file:
-                content = content_file.read()
-                if start_info in content:
-                    break
+            status_code = subprocess.getstatusoutput(
+                '{pg_ctl} status -D {datadir}'.format(
```

Comment:
use f-string

Reply:
Another question, can we simply rely on self.running method instead?

Reply:
only self.running is used now